lib/driver.js

```diff
@@ -100,13 +100,13 @@ class YouiEngineDriver extends BaseDriver {
             break;
           case "yimac":
             this.device = new YiMac();
-            this.device.startSession(caps);
+            await this.device.startSession(caps);
             break;
           case "bluesky":
             this.device = new BlueSky();
-            this.device.startSession(caps);
+            await this.device.startSession(caps);
             break;
-          case "yitvos":
+          case "yitvos": {
             let shell = require('shelljs');
             if (shell.exec(`instruments -s devices | grep '${caps.udid}'`).includes('(Simulator)')) {
```

Comment:
Can be one line, doesnt really matter.
if (shell.exec(`instruments -s devices | grep '${caps.udid}'`).includes('(Simulator)')) this.device = . new TvOsSimulator();

Reply:
expected bracket after if statement